README.md

```diff
@@ -523,6 +523,18 @@ var objectElement = minim.toElement({ foo: 'bar' });
 var keys = objectElement.keys() // ['foo']
 ```
 
+##### delete
```

Comment:
I think `remove` is a better name for this method as we're removing an element from the object. Deleting would imply the element is gone, but if I had a prior reference to it I can keep using the element. It is not "deleted".